airbyte-integrations/connectors/source-freshcaller/source_freshcaller/manifest.yaml

```diff
@@ -99,8 +99,6 @@ definitions:
       $ref: "#/definitions/retriever"
       requester:
         $ref: "#/definitions/requester"
-        request_parameters:
```

Comment:
Was this deprecated?

Reply:
https://github.com/airbytehq/airbyte/pull/35892/files#r1669085305